task-management/src/main/java/org/exoplatform/task/management/templates/detail.gtmpl

```diff
@@ -23,8 +24,21 @@
 	def commentCount = taskModel.getCommentCount();
 	def comments = taskModel.getComments();
 	def currentUser = taskModel.getCurrentUser();
-   def labels = taskModel.getLabels();
-   def editableClasses = "hoverStatus editableField editable";
+    def labels = taskModel.getLabels();
+    def editableClasses = "hoverStatus editableField editable";
+    def projectEditableClasses ="";
+    def projectDataToggle="";
+    def projectInlineEditClass="";
+    if (task.status == null && task.getCreatedBy() == identity.getUserId()) {
+        projectEditableClasses = editableClasses;
+        projectDataToggle = "tooltip";
+        projectInlineEditClass = "uiEditableInline";
+    }
+    if (task.status != null && task.status.project.canEdit(identity)) {
+        projectEditableClasses = editableClasses;
+        projectDataToggle = "tooltip";
+        projectInlineEditClass = "uiEditableInline";
```

Comment:
I just split the condition to give the user the ability the change  the task's project to make ti more clear,  
I'll update the PR with one block
`if ((task.status == null && task.getCreatedBy() == identity.getUserId())||(task.status != null && task.status.project.canEdit(identity))) `